docs/tracing/10-01-traces-troubleshooting.md

```diff
@@ -47,6 +47,6 @@ EOF
 
 ## Define the value in the Runtime
 
-If you have already installed Kyma and do not want to trigger any updates, edit the `istiod` deployment to set the desired value for **PILOT_TRACE_SAMPLING**. For detailed instructions, see the [Istio documentation](https://istio.io/latest/docs/tasks/observability/distributed-tracing/configurability/#customizing-trace-sampling).
+If you have already installed Kyma and do not want to trigger any updates, edit the `istiod` deployment to set the desired value for **PILOT_TRACE_SAMPLING**. For detailed instructions, see the [Istio documentation](https://istio.io/latest/docs/tasks/observability/distributed-tracing/configurability/mesh-and-proxy-config/#customizing-trace-sampling).
```

Comment:
One question, should we still link to it in the existing form if this method is deprecated or should this be rephrased/include a CAUTION note/...?
![image](https://user-images.githubusercontent.com/53601578/132006853-95b7cec4-ecaf-47de-b724-b77e88534936.png)